### PR TITLE
feat: add api 'GetFullSchemaType' to support external package in kcl doc

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [x] N
- [ ] Y 

#### 2. What is the scope of this PR (e.g. component or file name):

pkg/kcl/api.go

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [x] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

### CODE REVIEW

File: pkg/kcl/opt.go [hunk 1]
```diff
@@ -72,10 +73,6 @@ func ParseArgs(pathList []string, opts ...Option) (Option, error) {
 		}
 	}
 
-	if len(args.KFilenameList) == 0 {
```
Comment (Peefy) [inline pkg/kcl/opt.go]: Why remove the check here?

**Comment by Peefy** [comment]
cc @zong-zhe, I have upgrade the kcl artifact to v0.7.0-beta.2 and check passed. Ref:
+ https://github.com/kcl-lang/kcl-go/pull/191
+ https://github.com/kcl-lang/kcl-go/pull/189

After this PR has been merged, we can trigger a kcl-go v0.7.0-beta.2 tag for kpm. Then we can trigger a 0.5.0 kpm tag for kcl-go. After all, the v0.7.0 kcl-go tag release!

File: pkg/kcl/opt.go (hunk 1 above)
Comment (zong-zhe) [reply pkg/kcl/opt.go]: fixed. removed this changes

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**